test/unit/spec/EnrollCall_spec.js

```diff
@@ -589,6 +551,83 @@ function (Q, Okta, OktaAuth, LoginUtil, Util, Form, Beacon, Expect, $sandbox,
             ]);
           });
       });
+    }
+
+    Expect.describe('Enroll phone number', function () {
```

Comment:
does it have to use `Expect.describe`? `Expect.describe` has extra setups which is duplicated when embedded with another `Expect.describe`.

Reply:
@haishengwu-okta My intention was to keep the tests separate for IDX use cases, which I found useful while debugging the tests. Especially because i am reusing the test cases for idx and non-idx cases.